Rerank models can be started/updated from Trained Models page (#6556)
elastic/docs-content: explore-analyze/machine-learning/nlp/ml-nlp-rerank.md

    }
```

The API request automatically downloads and deploys the model. This example uses [autoscaling](../../../deploy-manage/autoscaling/trained-model-autoscaling.md) through adaptive allocation.

::::{note}
You might see a 502 bad gateway error in the response when using the {{kib}} Console. This error usually just reflects a timeout while the model downloads in the background. You can check the download progress in the **{{ml-app}}** UI. If using the Python client, you can set the `timeout` parameter to a higher value.
::::

### Using the {{models-app}} page [ml-nlp-rerank-deploy-kibana]

```{applies_to}
stack: ga 9.5
serverless: ga
```

You can also start or update a rerank model deployment from the **{{models-app}}** page:

1. Navigate to **{{ml-app}}** > **{{models-app}}** from the main menu, or use the [global search field](../../find-and-organize/find-apps-and-objects.md).
2. Find the rerank model (for example, `.rerank-v1`) in the list.
3. From the **Actions** menu, use **Start deployment** to deploy the model or **Update deployment** to change an existing deployment.

> Unlike embedding models, rerank models do not show the **Optimize for use case** selector (Ingest, Search, Balanced) in the deployment modal. This optimization applies only to embedding models.

Reviewer: What happens for rerank model is that they are optimized for search by default. That is why we do not see an option to optimize for ingest. I think this might be worth mentioning/ What do you think? More info [here](https://github.com/elastic/kibana/pull/261119)

Reviewer 2: "Rerank models are optimized for search by default" might be a simpler alternative to explaining the non-appearance of use case optimizer selector

Author: ```suggestion
Rerank models are optimized for search by default. This optimization applies only to embedding models.
```
